Makefile

```diff
@@ -3,36 +3,24 @@
 #
 # Not really important, but convenient.
```

Comment:
I think this comment (the whole 3 lines) doesn’t apply any more: the makefile is important, and I forget if sphinx build always rebuilds all files or not

Reply:
Will update.

Sphinx builds changed files by default - we can pass a flag to build all, however I'm not sure this would add anything beyond for local teststing of the build process itself. PEP 0 is built every time however.

Reply:
Have updated